### PR TITLE
Add support for Winbond W25Q64FV

### PR DESCRIPTION
**Pull request by djix123**

### CODE REVIEW

**Comment by skerr92** [comment]
Related to issue #3782 and PR #3824 

Everything seems in order, probably should add a description to the PR. @tannewt aggregate this with my PR?

File: ports/stm/boards/stm32f411ce_blackpill/pins.c [hunk 1]
```diff
@@ -35,5 +35,8 @@ STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_C14), MP_ROM_PTR(&pin_PC14) },
     { MP_ROM_QSTR(MP_QSTR_C13), MP_ROM_PTR(&pin_PC13) },
     { MP_ROM_QSTR(MP_QSTR_LED), MP_ROM_PTR(&pin_PC13) },
+
+    { MP_ROM_QSTR(MP_QSTR_SDA), MP_ROM_PTR(&pin_PB07) },
+    { MP_ROM_QSTR(MP_QSTR_SCL), MP_ROM_PTR(&pin_PB06) },
```
Comment (dhalbert) [inline ports/stm/boards/stm32f411ce_blackpill/pins.c:40]: Did you mean to add these pins in this PR? There is no `board.I2C()` defined?

**Review by dhalbert — COMMENTED** [review]
1. Question re SDA/SCL pins. 

2. I'm not sure; did you actually try the W25Q64FV on a board?

File: ports/stm/boards/stm32f411ce_blackpill/pins.c (hunk 1 above)
Comment (djix123) [reply ports/stm/boards/stm32f411ce_blackpill/pins.c:40]: 1. The SDA/SCL pins commit looks like it got added to the pull request after I'd created it when I added those to my fork - wasn't the intention that they be part of this pull
2. Yes, tested W25Q64FV working with build of 6.1.0-beta.2 and head.

Comment (dhalbert) [reply ports/stm/boards/stm32f411ce_blackpill/pins.c:40]: could you push a new commit that removes the SDA and SCL pins?

Comment (djix123) [reply ports/stm/boards/stm32f411ce_blackpill/pins.c:40]: Ok - will try that

**Comment by djix123** [comment]
Resubmitting with only W25Q64FV commit

**Comment by dhalbert** [comment]
Clean pull request is fine, but not required. I'll use the other one, thanks!

**Pull request closed without merging**